src/includes/configuration/auto-session-tracking/react-native.mdx

```diff
@@ -1,7 +1,28 @@
 To benefit from the health data, you must use at least version 1.4.0 of the React Native SDK and enable the collection of the release health data in the initialization options of the SDK.
 
+We mark the session as crashed if an _unhandled error_ or _unhandled promise rejection_ bubbled up to the global handler.
```

Comment:
Off topic of this docs PR, but is this true? unhandled promise rejections don't crash the app in RN and just fail silently, would make more sense to mark the session as errored rather than crashed. I'm not sure where this marking is being done @bruno-garcia

Reply:
@jennmueng what's the `event.mechanism.handled` value for an unhandled promise from RN events?
if it's `false`, on Android, we'd mark it as `crashed`.
on IOS, it's different, see https://github.com/getsentry/sentry-cocoa/pull/906

fwiw, on Flutter, we never mark as crashed unless there's a native crash (C/C++) code otherwise it's always errored, Flutter never crashes on the Dart layer, but RN does, right?

Reply:
@marandaneto I see, yes on RN an unhandled error on the JS engine would crash the app.

I just checked, `event.exception.mechanism.handled` is true (although `event.mechanism` is not defined), and the level is `error`. So the session wouldn't be marked as crashed then @PeloWriter

Reply:
its `event.exception.mechanism.handled`, it was a typo, thanks for checking this out.

Reply:
> So the session wouldn't be marked as crashed then @PeloWriter

for _unhandled error_ or _unhandled promise rejection_, but for native crashes (Android and iOS) bits, they will be marked as Crashed.

Reply:
Although note as a user, I would expect an unhandled promise rejection to be `handled: false` though. Would give me the wrong idea if an unhandled error is marked as handled.

Reply:
@jennmueng the idea of `handled: false` for Mobile is, `did my App fully crashed/terminate?` so if it doesn't, it does make sense to be `handled: true`

Reply:
Hmm wouldn't `level: fatal` make more sense for that use?

Reply:
that's how the protocol was designed

Reply:
Updating to:

> We mark the session as an error if the SDK captures an event that contains an exception (this includes manually captured errors) for _unhandled error_ or _unhandled promise rejection_, but for native crashes (Android and iOS), they will be marked as crashed.

Reply:
This would correct:
> We mark the session as errored if the SDK captures an event that contains an exception (this includes manually captured errors) or a unhandled promise rejection, but for unhandled errors and native crashes (Android and iOS), they will be marked as crashed.

Reply:
Updated to above. Thanks.